contrib/python/podman/podman/libs/containers.py

```diff
@@ -201,7 +194,8 @@ def unpause(self):
             podman.UnpauseContainer(self._id)
             return self._refresh(podman)
 
-    def update_container(self, *args, **kwargs):
+    def update_container(self, *args, **kwargs):          \
```

Comment:
wassat ``\`` at the end?  It looks weird.

Reply:
trailing slash escapes newline, pylint complains if comment not on same line but also complains that line is too long. So I have to make code more complex to silence the code checker. :(